### PR TITLE
Tune verbosity of search value conflicts

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #26923

@aprilnovak

### CODE REVIEW

File: framework/src/transfers/MultiAppGeneralFieldTransfer.C [hunk 1]
```diff
@@ -1294,29 +1325,33 @@ MultiAppGeneralFieldTransfer::outputValueConflicts(
...
 
   if (_local_conflicts.empty() && _received_conflicts.empty())
   {
     if (isParamSetByUser("search_value_conflict"))
       mooseInfo("Automated diagnosis did not detect floating point indetermination in transfer");
-    else if (_to_problems.size() > 10 || _from_problems.size() > 10)
-      mooseInfo("Automated diagnosis did not detect any floating point indetermination in "
-                "transfer. You may consider turning it off using `search_value_conflicts=false` "
-                "to improve performance/scalability.");
+    else if (_to_problems.size() > 10 || _from_problems.size() > 10 || _communicator.size() > 10)
```
Comment (lindsayad) [inline framework/src/transfers/MultiAppGeneralFieldTransfer.C:1350]: Do you think only outputting for > 10, is better than outputting > 1? It's not a leading question; I want to hear your thinking on it

**Review by lindsayad — CHANGES_REQUESTED** [review]

File: framework/src/transfers/MultiAppGeneralFieldTransfer.C (hunk 1 above)
Comment (GiudGiud) [reply framework/src/transfers/MultiAppGeneralFieldTransfer.C:1350]: If we have every run output "please turn me off" the default input will just turn it off. When it s useful stuff
10 is arbitrary though. Maybe the threshold for the cost being unbearable is 2 for some sims, maybe it s 100

**Review by lindsayad — APPROVED** [review]

**Comment by GiudGiud** [comment]
Test failures are in other apps and are not related to this, but to the TM rename

**Comment by GiudGiud** [comment]
Thanks for the review!

**Pull request merged**